coding/experiments/nyt/repeated_n_all_different/experiment.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import os
+import openai
+from pdb import set_trace as breakpoint
+import pickle
+from tqdm import tqdm
+import numpy as np
+# add src to path
+import sys
+sys.path.append('src')
+# import templatizing script
+from templatize import Templatizer
+
+# create templatizer
+templatizer = Templatizer(dataset_name='nytimes')
+# do for 10 runs
+n_runs = 10
+n_examples = 31
+
+for run in range(n_runs):
+    for n in range(31):
+        # get today's date as string MM-DD-YYYY
+        today = pd.to_datetime('today').strftime('%m-%d-%Y')
+        exp_dir = os.path.join(f'experiments/nyt/repeated_n_all_different/{today}/fewshot/{n}/run{run}')
```

Comment:
I'd like `coding/experiments/nyt/repeated_n_all_different/date/result` to be   `coding/experiments/nyt/date/repeated_n_all_different/result`

Reply:
Done!